sfs/mono/drivingfunction.py

```diff
@@ -282,7 +478,32 @@ def wfs_25d_focused(omega, x0, n0, xs, ns, xref=[0, 0, 0], c=None,
 
 
 def wfs_25d_preeq(omega, omalias, c):
-    """Preqeualization for 2.5D WFS."""
+    r"""Pre-equalization filter for 2.5-dimensional WFS.
+
+    Parameters
+    ----------
+    omega : float
+        Angular frequency.
+    omalias: float
+        Angular frequency where spatial aliasing becomes prominent.
+    c : float
+        Speed of sound.
+
+    Returns
+    -------
+    float
+        Complex weight for given angular frequency.
+
+    Notes
+    -----
+    .. math::
+
+        H(\w) = \begin{cases}
+            \sqrt{\i \wc} & \text{for } \w <= \w_\text{alias} \\
```

Comment:
See https://github.com/sfstoolbox/sfs-python/pull/104#discussion_r264035527

Reply:
missed that, thanks for the hint again, now fixed.